addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties

```diff
@@ -73,7 +73,7 @@ pscanrules.informationdisclosurereferrer.issuer.field=Issuer:
 
 pscanrules.informationdisclosuresuspiciouscomments.name=Information Disclosure - Suspicious Comments
 pscanrules.informationdisclosuresuspiciouscomments.desc=The response appears to contain suspicious comments which may help an attacker. Note: Matches made within script blocks or files are against the entire content not only comments.
-pscanrules.informationdisclosuresuspiciouscomments.otherinfo=The following comment/snippet was identified via the pattern: {0}\n{1}
+pscanrules.informationdisclosuresuspiciouscomments.otherinfo=The following comment/snippet was identified via the pattern: {0}
```

Comment:
This needs to be adjusted, there's no longer a comment/snippet. Mention the evidence?

Reply:
Fixed, hope it is better now